build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -111,35 +132,41 @@ export default class MergeBranch extends BaseCommand<typeof MergeBranch> {
 			flags.source,
 		);
 
-		this.log(`Unmerged commit list: ${unmergedCommitList}`);
-
 		if (unmergedCommitList.length === 0) {
 			this.log(
 				chalk.green(
 					`${flags.source} and ${flags.target} branches are in sync. No commits to merge`,
 				),
 			);
-			return;
+			this.exit(0);
 		}
```

Comment:
why is this.exit(0) added here?

Reply:
It's clearer than return. They're basically the same in this case, but exit makes it clearer that the command is
exiting. Compare to returning from the init() method. You wouldn't expect that to exit the command; similarly you
wouldn't expect run() to exit the command upon return either, so this makes it explicit.